### DISCUSSION TITLE
Restoring RLink2 after black screen

### DISCUSSION THREAD

**Discussion by lopezcelani** [cedricp/ddt4all#1255] [Q&A]
After many attempts and a long wait, I was finally able to bring my R-Link2 screen back to life, possibly after accidentally (or maybe not) pressing the HIGH button in Developer Mode.

However, a few features stopped working after this, such as the climate controls, parking sensors, and the temperature display (which now shows --F°).

I'm currently running R-Link2 version 2.2.19.300 with boot 5615.

Any advice? I’ve already been to a Renault garage, but they don’t have any newer versions available in my area.

**Comment by matzel82** [comment]
Dont touch it again… let it run.
